cherrypy/test/sessiondemo.py

```diff
@@ -2,7 +2,7 @@
 """A session demonstration app."""
 
 import calendar
-from datetime import datetime
+import datetime
```

Comment:
I think, I'd prefer a scoped "import from" variant, because reading things like `datetime.datetime` is rather confusing.
```suggestion
from datetime import datetime, UTC
```

https://youtu.be/x-kB2o8sd5c?t=1038&feature=shared nicely explains when this might be better.

Reply:
done

Reply:
Thanks! Marking this as resolved!